### DISCUSSION TITLE
为什么 input 组件的 slot:right 的 font-size 要设置成 0， 是不小心打错了？

### DISCUSSION THREAD

**Discussion by GreatAuk** [jdf2e/nutui#2862] [Q&A]
https://github.com/jdf2e/nutui/blob/9745bb09568c0e0a7988155edbcffc0d0b5d5012/src/packages/__VUE/input/index.scss#L103C4-L103C18

<img width="698" alt="image" src="https://github.com/jdf2e/nutui/assets/20253809/1bf25a3e-a09f-4322-ab13-d888c7e5e94d">

**Comment by eiinu** [comment]
下个版本会修复，感谢反馈
